modules/go-setting-environment-annotations.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// * /gitops/health-information-for-resources-deployment.adoc
+
+:_content-type: REFERENCE
+[id='go-setting-environment-annotations_{context}']             
+= Setting up environment labels and annnotations
+
+[discrete]
+== Environment labels:
```

Comment:
```suggestion
== Environment labels
```
Per ISG: "Do not end a heading with a period or colon"

https://www.ibm.com/docs/en/ibm-style?topic=format-headings#punctuation